dashboard/src/components/Catalog/Catalog.tsx

```diff
@@ -9,7 +9,7 @@ import { MessageAlert } from "../ErrorAlert";
 import LoadingWrapper from "../LoadingWrapper";
 import PageHeader from "../PageHeader";
 import SearchFilter from "../SearchFilter";
-import CatalogItem, { ICatalogItem } from "./CatalogItem";
+import CatalogItem, { ICatalogItemProps, IChartCatalogItem, IOperatorCatalogItem } from "./CatalogItem";
```

Comment:
The specific interfaces are only required for generating the segment of the key that differs depending on the type... not sure if that segment is required for uniqueness? (see below, line 106, where it includes either the csv or the repo name).